### PR TITLE
Hotfix for #956 - Fix for Feature Selection Hang on MapView Gizmo (#956)

### PR DESCRIPTION
**Pull request by swainn**
See #956 
* Fix for WMS feature selection infinite loop
* Remove deprecated Overlay options

### CODE REVIEW

**Review by shawncrawley — APPROVED** [review]
I did leave a comment about a `console.log` statement, but since it's no big deal I approve regardless.

File: tethys_sdk/static/tethys_sdk/js/utilities.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ function compute_center(features) {
                 let line_string = line_strings[i];
 
                 for (var j = 0; j < line_string.length; j++) {
+                    console.log(i, j)
```
Comment (swainn) [inline tethys_sdk/static/tethys_sdk/js/utilities.js:82]:
```suggestion
```

[the remark this answers is not in the document]
Comment (swainn) [reply tethys_sdk/static/tethys_sdk/js/utilities.js]: Done. Patched in main too.

[the remark this answers is not in the document]
Comment (swainn) [reply tethys_sdk/static/tethys_sdk/js/utilities.js]: Done. Patched in main too.

**Pull request merged**